doc/user/content/sql/system-catalog.md

```diff
@@ -197,6 +401,41 @@ Field       | Type       | Meaning
 `name`      | [`text`]   | The name of the object.
 `type`      | [`text`]   | The type of the object: either `table`, `source`, `view`, `sink`, or `index`.
 
+### `mz_peek_active`
+
+The `mz_peek_active` source describes all read queries ("peeks") that are
+pending in the dataflow layer.
+
+Field    | Type       | Meaning
+---------|------------|--------
+`uuid`   | [`text`]   | The ID of the connection that requested the peek. `uuid` is a misnomer; connection IDs are 32-bit unsigned integers.
```

Comment:
If it is a misnomer should we change it? I think this is (or was) the term in the code, but if we have some better name for it we can fix it.

Reply:
Yeah 100%; I wrote all this stuff down in #7174.